client/index.js

```diff
@@ -1,11 +1,10 @@
 import React from "react";
-import { render } from "react-dom";
+import ReactDOM from "react-dom";
 import App from "./App.jsx";
-import { BrowserRouter as Router } from "react-router-dom";
 
-render(
-  <Router>
```

Comment:
Are we getting rid of router?...I'm not complaining

Reply:
We have router in App.jsx

Reply:
cool